### DISCUSSION TITLE
ColorPicker not getting rendered

### DISCUSSION THREAD

**Discussion by silent21century** [AvaloniaUI/Avalonia#15851] [Q&A] [answered]
i have this problem with colorpicker control.
 was visible before in my project but after i remade the project with a new solution in rider, the colorpicker is not visible at all, i thought the problem was with rider 
so i remade a new project using command `dotnet new avalonia.mvvm -o GetStartedApp`  and only added to it colorpicker package at first and added it on the mainwindow with a small text, but after running it i got the same issue color  picker doesn't appear,
`note : when running the command you start with avalonia 11.0.10, i tried it and also tried 11.1.0-beta1 and 11.1.0-beta2 but the same issue` 
i also noticed that even without adding the colorpicker package and yet including an element from it to the project it doesn't show an error which is weird,
this is the code :
``` axaml
<Window xmlns="https://github.com/avaloniaui"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:vm="using:GetStartedApp.ViewModels"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        mc:Ignorable="d"
        d:DesignWidth="800"
        d:DesignHeight="450"
        x:Class="GetStartedApp.Views.MainWindow"
        x:DataType="vm:MainWindowViewModel"
        Icon="/Assets/avalonia-logo.ico"
        Title="GetStartedApp">
    <Design.DataContext>
        <!-- This only sets the DataContext for the previewer in an IDE,
             to set the actual DataContext for runtime, set the DataContext property in code (look at App.axaml.cs) -->
        <vm:MainWindowViewModel />
    </Design.DataContext>
    <StackPanel>
        <TextBlock Text="hi" />
        <ColorPicker />
        <TextBlock Text="{Binding Greeting}"
                   HorizontalAlignment="Center"
                   VerticalAlignment="Center" />
    </StackPanel>
</Window>
```

**Comment by rabbitism** [comment, marked as answer]
don't forget to reference the style

https://docs.avaloniaui.net/docs/reference/controls/colorpicker/#using-in-your-app

**Reply by silent21century** [reply]
that solved it, thanks a lot
